packages/private/edtr-io/src/hooks.ts

```diff
@@ -0,0 +1,20 @@
+import { selectors, useStore } from '@edtr-io/core'
```

Comment:
Probably no longer needed after integrating it completely?

Reply:
Is it ok to keep it till edtr.io is integrated completely?

Reply:
yep